fvtest/porttest/omrfileTest.cpp

```diff
@@ -3461,71 +3461,78 @@ TEST_F(PortFileTest2, file_test27)
 
 	reportTestEntry(OMRPORTLIB, testName);
 
-	omrfile_chmod(localFilename, 0777);
-	omrfile_unlink(localFilename);
-	fd1 = FILE_OPEN_FUNCTION(OMRPORTLIB, localFilename, EsOpenCreate | EsOpenWrite, 0666);
-	if (-1 == fd1) {
-		outputErrorMessage(PORTTEST_ERROR_ARGS, "%s() failed\n", FILE_OPEN_FUNCTION_NAME);
-	} else {
-		(void)FILE_CLOSE_FUNCTION(OMRPORTLIB, fd1);
-		for (m = 0; m < nModes; ++m) {
+#if !defined(OMR_OS_WINDOWS)
+	/* Do not run following test as a root user (euid == 0) as root can bypass file permissions. */
+	if (0 != omrsysinfo_get_euid()) {
+#endif /* !defined(OMR_OS_WINDOWS) */
```

Comment:
the `ifdef` at the closing brace can be avoided.
```suggestion
#if !defined(OMR_OS_WINDOWS)
	/* Do not run following test as a root user (euid == 0) as root can bypass file permissions. */
	if (0 != omrsysinfo_get_euid())
#endif /* !defined(OMR_OS_WINDOWS) */
	{
```